src/components/UserMenu/UserMenu.js

```diff
@@ -29,31 +33,36 @@ const UserMenu = () => {
         "is-expanded": isExpanded
       })}
     >
-      <div
-        className="user-menu__header"
-        onClick={() => setIsExpanded(!isExpanded)}
-      >
-        <i className="p-icon--user"></i>
-        {/* @TODO Add user name here */}
-        <span className="user-menu__name">Account</span>
-        <i className="p-icon--contextual-menu"></i>
-      </div>
-      <ul className="user-menu__options">
-        <li>
-          <Link className="user-menu__link" to="/settings">
-            Settings
-          </Link>
-        </li>
-        <li>
-          <Link
-            className="user-menu__link"
-            to="/"
-            onClick={() => dispatch(logOut(getState))}
+      {activeUser && (
```

Comment:
I'm not sure it's a great idea to gate on this value. I don't think we want to not render the entire user menu if their name isn't defined. A better approach might be to always render it but then have `activeUser` fall back to "Account" or something.

Reply:
The problem with that is you then get a flicker of "Account" on page load. When would a user login and not have a username?

Reply:
Ahh I see, the issue is that it might be possible for the `identity` for the user to not be defined depending on the identity provider. It's not likely, but if so then they are prevented from logging out in this case. 

If you'd like you can just leave it an empty string then, that way no name will be shown and no flash, but the menu will still work.